tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -3552,4 +3553,126 @@ public function provide_test_get_excluded_payment_method_types(): array {
 			],
 		];
 	}
+
+	/**
+	 * Test that payment_scripts() only enqueues Stripe JS on product pages when ECE is disabled.
+	 */
+	public function test_payment_scripts_enqueues_only_stripe_js_on_product_page_when_ece_disabled() {
+		$product = WC_Helper_Product::create_simple_product();
+		$this->go_to( get_permalink( $product->get_id() ) );
+
+		$stripe_settings                                       = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WC_Stripe_Helper' in method 'test_payment_scripts_enqueues_only_stripe_js_on_product_page_when_ece_disabled'. (undefined)

(StaticAccess)